### PR TITLE
added edge as a browser compatibility and added TEMPLATE.md

### PR DESCRIPTION
**Pull request by jwallet**

### CODE REVIEW

**Comment by cht8687** [comment]
please help resolve the conflicts so that we can merge.

File: README.md [hunk 1]
```diff
@@ -222,9 +222,9 @@ Creates an array with all falsy values removed.
 
 #### Browser Support
```
Comment (stevemao) [inline README.md]: Should this be 

```md
#### Browser Support for `array.prototype.filter`
```?

Comment (jwallet) [reply README.md]: yep

File: README.md [hunk 2]
```diff
@@ -2122,9 +2152,9 @@ Removes all duplicates entries from an array.
...
```
Comment (stevemao) [inline README.md]: Browser Support for `new Set()`?

**Comment by jwallet** [comment]
should we merge it ?

**Comment by jwallet** [comment]
i also fixed all browser icons that got their path changed from `raw.github.com` to `raw.githubusercontent.com`

**Comment by jwallet** [comment]
reordered readme

File: README.md [hunk 3]
```diff
@@ -111,59 +111,67 @@ and will not work with objects. If this functionality is needed,
...
 1. [_.after](#_after)
-1. [_.bind](#_bind)
-1. [_.partial](#_partial)
+2. [_.bind](#_bind)
+3. [_.partial](#_partial)
 
 **[Lang](#lang)**
 
+<<<<<<< HEAD
+=======
```
Comment (stevemao) [inline README.md]: Sorry, looks like you created some conflicts

File: README.md [hunk 4]
```diff
@@ -111,59 +111,67 @@ and will not work with objects. If this functionality is needed,
...
-1. [_.orderBy](#_sortby-and-_orderby)
-1. [_.pluck](#_pluck)
-1. [_.range](#_range)
-1. [_.reduce](#_reduce)
-1. [_.reduceRight](#_reduceright)
-1. [_.size](#_size)
-1. [_.some](#_some)
-1. [_.sortBy](#_sortby-and-_orderby)
-1. [_.uniq](#_uniq)
+2. [_.every](#_every)
```
Comment (stevemao) [inline README.md]: if you just write `1.` it will auto create an order list with correct numbers.

**Comment by stevemao** [comment]
Thanks for this @jwallet. once conflicts are solved we can merge and iterate on it.

File: README.md (hunk 4 above)
Comment (jwallet) [reply README.md]: yeah sorry markdown vscode extension, will disable it

**Comment by jwallet** [comment]
should we merge it ?

File: README.md [hunk 5]
```diff
@@ -1806,63 +1868,16 @@ Retrieves all the names of the object's own enumerable properties.
...
-![Chrome][chrome-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
-:-: | :-: | :-: | :-: | :-: |
-  54 ✔  | 47 ✔ |  Not supported  |  41.0 ✔ |  10.1 ✔ |
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: |  :-: |
+  5.0 ✔  | ✔ | 4.0 ✔ |  9.0 ✔ |  12.0 ✔ |  5.0 ✔ |
 
 **[⬆ back to top](#quick-links)**
 
+<<<<<<< HEAD
```
Comment (stevemao) [inline README.md]: Still got conflicts, otherwise looks good

**Pull request merged**